test/rmq/test_coroutine_communicator.py

```diff
@@ -197,3 +197,12 @@ def broadcast_subscriber(_comm, _body, _sender=None, _subject=None, _correlation
 
 
 # endregion
+
+
+@pytest.mark.asyncio
+async def test_server_properties(communicator: kiwipy.rmq.RmqCommunicator):
+    props = communicator.server_properties
+    assert isinstance(props, dict)
+
+    assert props['product'] == b'RabbitMQ'
```

Comment:
Does it reliably also give the version number across platforms? I guess that is really what we are interested in, in `aiida-core`.

Reply:
I'm not sure what you mean by 'across platforms' but the RMQ protocol states that the server SHOULD give a version number:

https://www.rabbitmq.com/amqp-0-9-1-reference.html#connection.start.server-properties

I've added this to the docstring...I think we can rely on it.

Reply:
Yeah, that's what I thought (re: platforms).  Indeed, to the extent that RMQ conforms to its own spec it should be platform agnostic in this regard...